static/dream_web/index.html

```diff
@@ -65,6 +65,9 @@ <h2 id="header">Stable Diffusion Dream Server</h2>
           <span>&bull;</span>
           <button type="button" id="reset-all">Reset to Defaults</button>
           <br>
+          <label for="progress_images">Display in-progress images (slows down generation):</label>
+          <input type="checkbox" name="progress_images" id="progress_images">
```

Comment:
Quick consistency nitpick, can this be named progress-images?

Reply:
Sure. The underscore was just for consistency with `gfpgan_strength` immediately below.

Reply:
Actually, this file seems to consistently use underscores for names, so I've left it as-is for now.

Reply:
@TesseractCat are there still issues, or are we good to go? (I've re-requested a review. Hope that's etiquette.)

Reply:
@lstein could this get merged as-is, and then @TesseractCat can submit any further style nits in a followup PR? I'd like to stop needing to rebase this PR, and to get https://github.com/lstein/stable-diffusion/pull/182 in.